pyrft/random_field_generation.py

```diff
@@ -21,6 +21,11 @@ def smooth(data, fwhm, mask = 0):
 
   Examples
   ---------------------
+# 1D example

```

Comment:
please fix the indentation

Reply:
See my comment above about the examples section.